sdk/python/kfp/client/client.py

```diff
@@ -303,9 +303,9 @@ def _load_config(
                 namespace)
         return config
 
-    def _is_inverse_proxy_host(self, host):
+    def _is_inverse_proxy_host(self, host: str) -> bool:
         if host:
-            return re.match(r'\S+.googleusercontent.com/{0,1}$', host)
+            return bool(re.match(r'\S+.googleusercontent.com/{0,1}$', host))
```

Comment:
`re.match` does not return a bool, but it does return a truthy object if a match is found, and `None` (falsey) if a match is not found. Let me know if you want to be more explicit about this, e.g.:
```
if re.match(r'\S+.googleusercontent.com/{0,1}$', host) is None:
    return False
```

Reply:
Perhaps it makes sense to expand with an explicit `None` check, since it does not require future readers to know about the truthiness/falsiness properties of `re.Match` objects.